CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 Change Log
 ==========
+Version 0.3.0 *(2019-02-04)*
+----------------------------
+
+* Prefill attachments  
+
+Bug Fixes:
+
+* Ensured all places the Beacon Config is accessed it has the overrides applied 
```

Comment:
I don't follow this one - can we reword it to be less about what we did, and more about the benefit?  I would reword it but I don't know what benefit we are providing to developers.  Is it something like 'You can now override every Beacon Builder setting in the SDK by doing X, Y and Z' (the config is determined by Builder, so lets refer to that if that is what we mean)